docs/compact_ranges.md

```diff
@@ -210,3 +210,37 @@ the load on the coordinator with the load on "leaf" workers.
 
 The described approach can be used to implement a scalable tamper-evident log
 based on Merkle trees.
+
+### Updatable Proofs
+
+Logs based on Merkle trees are rarely static - usually they grow in append-only
+fashion. Proofs built for one state of the tree get outdated with later states.
+If proofs are made using compact ranges, they can be updated by merging in the
+compact range between the previous and the next tree size.
+
+For example, an inclusion proof for range `[L, R)` in a tree of size `N` can be
+represented with compact ranges `[0, L)` and `[R, N)`, as described in a
+[section](#arbitrary-inclusion-proofs) above. When the tree moves to a state of
+size `N+delta`, it is possible to update this inclusion proof by merging the
+compact range `[R, N)` with `[N, N+delta)`.
+
+Updating a proof incrementally is more efficient in terms of space / bandwidth
+than fetching an entire proof each time. For example, a consistency proof in a
+tree up to size `N` takes `O(log N)` space, and fetching it `O(N)` time takes
```

Comment:
/tree up to size N/tree of size N/?

Reply:
The idea is that the tree grows gradually to size `N`, I was avoiding fixing the size.

Reworded a bit, how does it look?

Reply:
Yeah, that's clearer - ta!